config/samples/flows_v1alpha1_flowcollector.yaml

```diff
@@ -29,7 +30,7 @@ spec:
     logLevel: info
     enableKubeProbes: true
     healthPort: 8080
-    prometheusPort: 9102
+    prometheusPort: 9090
```

Comment:
I remember you mentioned this inconsistency.
I've just opened a PR to fix it, but the other way around, using 9102 in kubebuilder: https://github.com/netobserv/network-observability-operator/pull/149